components/ResearchCoin/TransactionCard.js

```diff
@@ -59,6 +65,34 @@ const TransactionCard = (props) => {
               ? "Withdrawal"
               : ""}
           </div>
+          {paper && (
+            <Link href={`/paper/${paper.id}/${paper.slug}`}>
+              <a
+                className={css(styles.metatext)}
+                onClick={(e) => {
+                  e.stopPropagation();
+                }}
+              >
+                <div className={css(styles.metatext)}>{paper.paper_title}</div>
+              </a>
+            </Link>
+          )}
+          {comment && (
+            <Link
+              href={`/paper/${comment.paper}/${comment.paper_slug}#comments`}
```

Comment:
I don't think this would work if there if it is a post or hypothesis comment

Reply:
oh right, let me check the type coming through